tests/config_check_test.ts

```diff
@@ -10,24 +10,12 @@ const subfolder = path.resolve(REPO_ROOT, "linters");
 describe("Global config health check", () => {
   const driver = setupDriver(subfolder, {
     setupGit: false,
-    setupTrunk: false,
-    launchDaemon: true,
+    setupTrunk: true,
+    launchDaemon: false,
   });
 
   // Step 2: Validate config
   it("trunk config print from repo root", async () => {
-    // trunk-ignore(eslint/@typescript-eslint/no-unsafe-assignment)
```

Comment:
Is the reason we're doing this because we want to do the driver matrix testing at the workflow level?